make-indices.py

```diff
@@ -83,10 +83,88 @@ def make_cluster_index(sizes):
     np.save('gmsc-db/GMSC10.cluster.index.npy', ix)
     np.save('gmsc-db/GMSC10.cluster.data.npy', data)
 
+@TaskGenerator
+def create_tax_index(infile,outfile1,outfile2):
+    import lzma
+    tax_set = set()
+    n = 0
+    with lzma.open(infile,'rt') as f:
+        for line in f:
+            linelist = line.strip().split('\t')
+            if len(linelist) == 2:
+                tax_set.add(linelist[1])
+            else:
+                tax_set.add('Unknown')
+            
+    tax_order = sorted(list(tax_set))
+    with open(outfile1,'wt') as out:
+        for item in tax_order:
```

Comment:
This could be

```python
for n,item in enumerate(tax_order):
   ...
```

instead of setting `n = 0` in line `90` and `n += 1` in `103`